pkg/expect/expect.go

```diff
@@ -37,6 +38,11 @@ var (
 	ErrProcessRunning = fmt.Errorf("process is still running")
 )
 
+type ExpectedResponse struct {
+	Value         string
+	IsRegularExpr bool
```

Comment:
Do you want to add some regex tests?

Reply:
added, please take a look at the second commit. thx

Reply:
Thanks.